src/styles/ProjectDetailPage.scss

```diff
@@ -5,6 +5,9 @@
         width: 100%;
         z-index: 2
     }
+    .GridProjectDetailPage {
+        margin-bottom: 2rem;
+    }
```

Comment:
Suggestion:

```
> .MuiGrid-item {
    margin-bottom: 2rem;
}
```

Reply:
If we use `.MuiGrid-item` this happens 
![image](https://user-images.githubusercontent.com/86666889/148819522-5fbb5329-679c-45bd-9e6b-19cf946b7b8e.png)